molecule/cloud-aws-direct/requirements.yml

```diff
@@ -0,0 +1,5 @@
+---
```

Comment:
We don't need requirements file in scenario folder as long as we have it in root.